docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -58,6 +59,17 @@ the implementation without compat concerns in future releases.
   - Add new test code following [conventions](project-guidelines.md#code-file-naming-conventions) for new files to that are specific to the new target framework.
   - To run just the new test configuration run `dotnet msbuild <Library>.csproj /t:RebuildAndTest /p:TargetGroup=<TargetGroup>`
 
+## Documentation
+
+New public APIs must be documented with triple slash comments on top of them. Visual Studio automatically generates the structure for you when you type `\\\`.
+
+If your new API or the APIs it calls throw any exceptions, those need to be manually documented by adding the `<exception></exception>` elements.
+
+After your change is merged, we will eventually port them to the dotnet-api-docs repo, where we will review them for language and proper style (See the [Docs contributing guide](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md) for more information).
+
+Once the dotnet-api-docs change is merged, your comments will start showing up in MS Docs, and later in IntelliSense.
```

Comment:
```suggestion
Once the dotnet-api-docs change is merged, your comments will start showing up in the official API documentation at http://docs.microsoft.com/, and later it will appear in IntelliSense in Visual Studio and Visual Studio Code.
```